APCAppCore/APCAppCore/Library/ScheduleImporter/APCScheduleImporter.m

```diff
@@ -66,7 +66,7 @@
  Controls whether we compute and show (very) detailed debugging
  printouts.  This will always be NO in a release build.
  */
-static BOOL const kAPCShowDebugPrintouts = YES;
+static BOOL const kAPCShowDebugPrintouts = NO;
```

Comment:
Yes.  The printouts are very verbose.  I wanted them to be available to anyone who needs them, but not get in the way unless asked for.